README.md

```diff
@@ -143,15 +142,15 @@ an uber JAR file that contains all dependencies required to run the pipeline on
 
 Creating an uber jar for Pramen is very easy. Just clone the repository and run one of the following commands:
 ```sh
-sbt -DSPARK_VERSION=2.4.8 ++2.11.12 assembly 
+sbt -DSPARK_VERSION="2.4.8" ++2.11.12 assembly 
 
-sbt -DSPARK_VERSION=2.4.8 ++2.12.17 assembly 
-sbt -DSPARK_VERSION=3.0.3 ++2.12.17 assembly 
-sbt -DSPARK_VERSION=3.1.3 ++2.12.17 assembly 
-sbt -DSPARK_VERSION=3.2.2 ++2.12.17 assembly 
+sbt -DSPARK_VERSION="2.4.8" ++2.12.17 assembly 
+sbt -DSPARK_VERSION="3.0.3" ++2.12.17 assembly 
+sbt -DSPARK_VERSION="3.1.3" ++2.12.17 assembly 
+sbt -DSPARK_VERSION="3.2.2" ++2.12.17 assembly 
 
-sbt -DSPARK_VERSION=3.2.2 ++2.13.10 assembly
-sbt -DSPARK_VERSION=3.3.1 ++2.13.10 assembly 
+sbt -DSPARK_VERSION="3.2.2" ++2.13.10 assembly
```

Comment:
Unfortunately, on windows, it works only with these quotes 😢

Reply:
For some reason, (java? sbt?) quits parsing the property value as it encounters the dot 
```
PS C:\code\scala\pramen\pramen> sbt -DSPARK_VERSION=3.2.2 ++2.12.17 assembly -v
# Executing command line:
"C:\Users\AB0272B\.jdks\corretto-1.8.0_352\bin\java.exe"
-Dfile.encoding=UTF-8
-Xms1024m
-Xmx1024m
-Xss4M
-XX:ReservedCodeCacheSize=128m
-cp
"C:\Users\AB0272B\scoop\apps\sbt\current\bin\sbt-launch.jar"
xsbt.boot.Boot
-DSPARK_VERSION=3
.2.2
++2.12.17
assembly
```

Reply:
This is a very good spot! Thanks for fixing it.